pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -27,9 +27,15 @@ let
       cp libsha1.so $out/lib/
     '';
   };
+  hgSrc = if srcRev != null then
+    fetchhg {
+      url = srcUrl;
+      rev = srcRev;
+      sha256 = srcSha256;
+    } else null;
```

Comment:
This should just be the default without null checks.

Reply:
But we don't want to fetch anything by default, HOL is distributed with the release. That is just to allow the user to specify a development version of HOL for their installation